modules/data_release/README.md

```diff
@@ -1,16 +1,50 @@
-# Data Release Module
+# Data Release
 
-The Data Release Module can be used to easily distribute packaged data releases of your study.
 
-## Features:
+## Purpose
 
-- Upload new releases through the front-end web interface
-- Differentiate releases using (optionally unique) versioning tag
-- Assign permissions for each release to any specific user
+The Data Release module can be used to distribute packaged data releases 
+of the study to the analysis team. This module can be used by projects 
+which desires to publish based on the exact same set of data referred
+as data release. Along the lifespan of a study, several data releases might
+be made available as the dataset keeps growing.
 
-## Other notes:
 
-- Uploads are stored under the modules/data_release/user_uploads directory which can easily be symlinked to another location if necessary, but please create and make sure it is writable by your web server
-- Only superusers have permission to upload files and grant permissions
-- Remove permissions by deleting rows in the data_release_permissions table
-- Upload date will automatically be added during file upload
+## Intended users
+
+- Researchers and analysis teams
+- Data coordinator that will manage access to and upload of the data releases
+
+
+## Scope
+
+The module is used to manage, upload and download data releases of the study.
+- Different data releases can be differentiated using the version tag.
+- Data coordinators can upload data releases of the study via the front-end
+web interface.
+- For each data release, data coordinators can manage permissions of any
+  specific user.
+- Upload date will automatically be added during file upload.
+
+
+## Permissions
+
+- Only superusers can upload anything, no one else has upload permission.
+- Only superusers can grant permissions - whether by "version" or specific
+  "file_name", no one else has granting permissions.
+- Once a user is granted permission on any data release, they will be able
+  to see the data release and download it directly from the module.
+- Users permissions to view data releases are based on the granular file
+  level permissions in the `data_release_permissions` table. Granting by
+  version grants every file tied to the specified version.
+- At the moment, the only way to remove a user's permission to a specific data
```

Comment:
I don't know if this last point belongs under "permissions".. maybe under a different caveats section? Or in the scope? Or just as a Caveat/Note at the end of the list instead of as a point in the list?